2025-07-06 10:27:23,934 - asyncio - DEBUG - Using proactor: IocpProactor
2025-07-06 10:27:23,936 - httpx - DEBUG - load_ssl_context verify=True cert=None trust_env=True http2=False
2025-07-06 10:27:23,938 - httpx - DEBUG - load_verify_locations cafile='c:\\Users\\Rachid Aitali\\OneDrive - 4JM Solutions Limited\\Desktop\\SniperX-V3-Best-Version-at-now\\.venv\\lib\\site-packages\\certifi\\cacert.pem'
2025-07-06 10:27:23,947 - httpx - DEBUG - load_ssl_context verify=True cert=None trust_env=True http2=False
2025-07-06 10:27:23,947 - httpx - DEBUG - load_verify_locations cafile='c:\\Users\\Rachid Aitali\\OneDrive - 4JM Solutions Limited\\Desktop\\SniperX-V3-Best-Version-at-now\\.venv\\lib\\site-packages\\certifi\\cacert.pem'
2025-07-06 10:27:23,957 - __main__ - INFO - Telegram Manager Bot started. Send /menu to interact.
2025-07-06 10:27:23,958 - telegram.ext.ExtBot - DEBUG - Entering: send_message
2025-07-06 10:27:23,965 - httpcore.connection - DEBUG - connect_tcp.started host='api.telegram.org' port=443 local_address=None timeout=5.0 socket_options=None
2025-07-06 10:27:24,104 - httpcore.connection - DEBUG - connect_tcp.complete return_value=<httpcore._backends.anyio.AnyIOStream object at 0x000001BB989EF010>
2025-07-06 10:27:24,104 - httpcore.connection - DEBUG - start_tls.started ssl_context=<ssl.SSLContext object at 0x000001BB989A8340> server_hostname='api.telegram.org' timeout=5.0
2025-07-06 10:27:24,264 - httpcore.connection - DEBUG - start_tls.complete return_value=<httpcore._backends.anyio.AnyIOStream object at 0x000001BB989EEC80>
2025-07-06 10:27:24,265 - httpcore.http11 - DEBUG - send_request_headers.started request=<Request [b'POST']>
2025-07-06 10:27:24,267 - httpcore.http11 - DEBUG - send_request_headers.complete
2025-07-06 10:27:24,267 - httpcore.http11 - DEBUG - send_request_body.started request=<Request [b'POST']>
2025-07-06 10:27:24,267 - httpcore.http11 - DEBUG - send_request_body.complete
2025-07-06 10:27:24,268 - httpcore.http11 - DEBUG - receive_response_headers.started request=<Request [b'POST']>
2025-07-06 10:27:24,340 - httpcore.http11 - DEBUG - receive_response_headers.complete return_value=(b'HTTP/1.1', 200, b'OK', [(b'Server', b'nginx/1.18.0'), (b'Date', b'Sun, 06 Jul 2025 09:27:24 GMT'), (b'Content-Type', b'application/json'), (b'Content-Length', b'352'), (b'Connection', b'keep-alive'), (b'Strict-Transport-Security', b'max-age=31536000; includeSubDomains; preload'), (b'Access-Control-Allow-Origin', b'*'), (b'Access-Control-Allow-Methods', b'GET, POST, OPTIONS'), (b'Access-Control-Expose-Headers', b'Content-Length,Content-Type,Date,Server,Connection')])
2025-07-06 10:27:24,343 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot7707158318:AAE8rWa4Q8Z_3avql6K0HAmXaooZAReoNF0/sendMessage "HTTP/1.1 200 OK"
2025-07-06 10:27:24,345 - httpcore.http11 - DEBUG - receive_response_body.started request=<Request [b'POST']>
2025-07-06 10:27:24,345 - httpcore.http11 - DEBUG - receive_response_body.complete
2025-07-06 10:27:24,346 - httpcore.http11 - DEBUG - response_closed.started
2025-07-06 10:27:24,346 - httpcore.http11 - DEBUG - response_closed.complete
2025-07-06 10:27:24,348 - telegram.ext.ExtBot - DEBUG - Message(channel_chat_created=False, chat=Chat(first_name='Ghost', id=7049055170, last_name='Number1', type=<ChatType.PRIVATE>), date=datetime.datetime(2025, 7, 6, 9, 27, 24, tzinfo=<UTC>), delete_chat_photo=False, entities=(MessageEntity(length=5, offset=36, type=<MessageEntityType.BOT_COMMAND>),), from_user=User(first_name='SPRD', id=7707158318, is_bot=True, username='tizi91Bot'), group_chat_created=False, message_id=13017, supergroup_chat_created=False, text='SniperX Control Bot is online! Send /menu to see options.')
2025-07-06 10:27:24,348 - telegram.ext.ExtBot - DEBUG - Exiting: send_message
2025-07-06 10:27:24,349 - telegram.ext.ExtBot - DEBUG - Entering: get_me
2025-07-06 10:27:24,349 - httpcore.http11 - DEBUG - send_request_headers.started request=<Request [b'POST']>
2025-07-06 10:27:24,350 - httpcore.http11 - DEBUG - send_request_headers.complete
2025-07-06 10:27:24,350 - httpcore.http11 - DEBUG - send_request_body.started request=<Request [b'POST']>
2025-07-06 10:27:24,350 - httpcore.http11 - DEBUG - send_request_body.complete
2025-07-06 10:27:24,350 - httpcore.http11 - DEBUG - receive_response_headers.started request=<Request [b'POST']>
2025-07-06 10:27:24,512 - httpcore.http11 - DEBUG - receive_response_headers.complete return_value=(b'HTTP/1.1', 200, b'OK', [(b'Server', b'nginx/1.18.0'), (b'Date', b'Sun, 06 Jul 2025 09:27:24 GMT'), (b'Content-Type', b'application/json'), (b'Content-Length', b'243'), (b'Connection', b'keep-alive'), (b'Strict-Transport-Security', b'max-age=31536000; includeSubDomains; preload'), (b'Access-Control-Allow-Origin', b'*'), (b'Access-Control-Allow-Methods', b'GET, POST, OPTIONS'), (b'Access-Control-Expose-Headers', b'Content-Length,Content-Type,Date,Server,Connection')])
2025-07-06 10:27:24,512 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot7707158318:AAE8rWa4Q8Z_3avql6K0HAmXaooZAReoNF0/getMe "HTTP/1.1 200 OK"
2025-07-06 10:27:24,512 - httpcore.http11 - DEBUG - receive_response_body.started request=<Request [b'POST']>
2025-07-06 10:27:24,512 - httpcore.http11 - DEBUG - receive_response_body.complete
2025-07-06 10:27:24,513 - httpcore.http11 - DEBUG - response_closed.started
2025-07-06 10:27:24,513 - httpcore.http11 - DEBUG - response_closed.complete
2025-07-06 10:27:24,513 - telegram.ext.ExtBot - DEBUG - User(api_kwargs={'can_connect_to_business': False, 'has_main_web_app': False}, can_join_groups=True, can_read_all_group_messages=False, first_name='SPRD', id=7707158318, is_bot=True, supports_inline_queries=False, username='tizi91Bot')
2025-07-06 10:27:24,513 - telegram.ext.ExtBot - DEBUG - Exiting: get_me
2025-07-06 10:27:24,513 - telegram.ext.ExtBot - DEBUG - This Bot is already initialized.
2025-07-06 10:27:24,514 - telegram.ext.Updater - DEBUG - Updater started (polling)
2025-07-06 10:27:24,514 - telegram.ext.Updater - DEBUG - Start network loop retry bootstrap del webhook
2025-07-06 10:27:24,514 - telegram.ext.Updater - DEBUG - Deleting webhook
2025-07-06 10:27:24,514 - telegram.ext.ExtBot - DEBUG - Entering: delete_webhook
2025-07-06 10:27:24,514 - httpcore.http11 - DEBUG - send_request_headers.started request=<Request [b'POST']>
2025-07-06 10:27:24,515 - httpcore.http11 - DEBUG - send_request_headers.complete
2025-07-06 10:27:24,515 - httpcore.http11 - DEBUG - send_request_body.started request=<Request [b'POST']>
2025-07-06 10:27:24,515 - httpcore.http11 - DEBUG - send_request_body.complete
2025-07-06 10:27:24,515 - httpcore.http11 - DEBUG - receive_response_headers.started request=<Request [b'POST']>
2025-07-06 10:27:24,673 - httpcore.http11 - DEBUG - receive_response_headers.complete return_value=(b'HTTP/1.1', 200, b'OK', [(b'Server', b'nginx/1.18.0'), (b'Date', b'Sun, 06 Jul 2025 09:27:24 GMT'), (b'Content-Type', b'application/json'), (b'Content-Length', b'68'), (b'Connection', b'keep-alive'), (b'Strict-Transport-Security', b'max-age=31536000; includeSubDomains; preload'), (b'Access-Control-Allow-Origin', b'*'), (b'Access-Control-Allow-Methods', b'GET, POST, OPTIONS'), (b'Access-Control-Expose-Headers', b'Content-Length,Content-Type,Date,Server,Connection')])
2025-07-06 10:27:24,674 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot7707158318:AAE8rWa4Q8Z_3avql6K0HAmXaooZAReoNF0/deleteWebhook "HTTP/1.1 200 OK"
2025-07-06 10:27:24,675 - httpcore.http11 - DEBUG - receive_response_body.started request=<Request [b'POST']>
2025-07-06 10:27:24,675 - httpcore.http11 - DEBUG - receive_response_body.complete
2025-07-06 10:27:24,675 - httpcore.http11 - DEBUG - response_closed.started
2025-07-06 10:27:24,675 - httpcore.http11 - DEBUG - response_closed.complete
2025-07-06 10:27:24,677 - telegram.ext.ExtBot - DEBUG - True
2025-07-06 10:27:24,677 - telegram.ext.ExtBot - DEBUG - Exiting: delete_webhook
2025-07-06 10:27:24,677 - telegram.ext.Updater - DEBUG - Bootstrap done
2025-07-06 10:27:24,677 - telegram.ext.Updater - DEBUG - Waiting for polling to start
2025-07-06 10:27:24,677 - telegram.ext.Updater - DEBUG - Polling updates from Telegram started
2025-07-06 10:27:24,677 - telegram.ext.Updater - DEBUG - Start network loop retry getting Updates
2025-07-06 10:27:24,677 - telegram.ext.ExtBot - DEBUG - Entering: get_updates
2025-07-06 10:27:24,678 - telegram.ext.Application - INFO - Application started
2025-07-06 10:27:24,679 - httpcore.connection - DEBUG - connect_tcp.started host='api.telegram.org' port=443 local_address=None timeout=5.0 socket_options=None
2025-07-06 10:27:24,729 - httpcore.connection - DEBUG - connect_tcp.complete return_value=<httpcore._backends.anyio.AnyIOStream object at 0x000001BB98A371C0>
2025-07-06 10:27:24,730 - httpcore.connection - DEBUG - start_tls.started ssl_context=<ssl.SSLContext object at 0x000001BB989A8240> server_hostname='api.telegram.org' timeout=5.0
2025-07-06 10:27:24,888 - httpcore.connection - DEBUG - start_tls.complete return_value=<httpcore._backends.anyio.AnyIOStream object at 0x000001BB98A37070>
2025-07-06 10:27:24,890 - httpcore.http11 - DEBUG - send_request_headers.started request=<Request [b'POST']>
2025-07-06 10:27:24,891 - httpcore.http11 - DEBUG - send_request_headers.complete
2025-07-06 10:27:24,891 - httpcore.http11 - DEBUG - send_request_body.started request=<Request [b'POST']>
2025-07-06 10:27:24,892 - httpcore.http11 - DEBUG - send_request_body.complete
2025-07-06 10:27:24,893 - httpcore.http11 - DEBUG - receive_response_headers.started request=<Request [b'POST']>
2025-07-06 10:27:29,018 - httpcore.http11 - DEBUG - receive_response_headers.complete return_value=(b'HTTP/1.1', 409, b'Conflict', [(b'Server', b'nginx/1.18.0'), (b'Date', b'Sun, 06 Jul 2025 09:27:29 GMT'), (b'Content-Type', b'application/json'), (b'Content-Length', b'143'), (b'Connection', b'keep-alive'), (b'Strict-Transport-Security', b'max-age=31536000; includeSubDomains; preload'), (b'Access-Control-Allow-Origin', b'*'), (b'Access-Control-Expose-Headers', b'Content-Length,Content-Type,Date,Server,Connection')])
2025-07-06 10:27:29,019 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot7707158318:AAE8rWa4Q8Z_3avql6K0HAmXaooZAReoNF0/getUpdates "HTTP/1.1 409 Conflict"
2025-07-06 10:27:29,020 - httpcore.http11 - DEBUG - receive_response_body.started request=<Request [b'POST']>
2025-07-06 10:27:29,021 - httpcore.http11 - DEBUG - receive_response_body.complete
2025-07-06 10:27:29,021 - httpcore.http11 - DEBUG - response_closed.started
2025-07-06 10:27:29,021 - httpcore.http11 - DEBUG - response_closed.complete
2025-07-06 10:27:29,023 - telegram.ext.Updater - ERROR - Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
2025-07-06 10:27:29,024 - telegram.ext.Application - ERROR - No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\ext\_updater.py", line 628, in _network_loop_retry
    if not await action_cb():
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\ext\_updater.py", line 333, in polling_action_cb
    raise exc
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\ext\_updater.py", line 322, in polling_action_cb
    updates = await self.bot.get_updates(
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\ext\_extbot.py", line 553, in get_updates
    updates = await super().get_updates(
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\_bot.py", line 394, in decorator
    result = await func(self, *args, **kwargs)  # skipcq: PYL-E1102
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\_bot.py", line 3542, in get_updates
    await self._post(
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\_bot.py", line 482, in _post
    return await self._do_post(
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\ext\_extbot.py", line 335, in _do_post
    return await super()._do_post(
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\_bot.py", line 510, in _do_post
    return await request.post(
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\request\_baserequest.py", line 168, in post
    result = await self._request_wrapper(
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\request\_baserequest.py", line 327, in _request_wrapper
    raise Conflict(message)
telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
2025-07-06 10:27:30,045 - telegram.ext.ExtBot - DEBUG - Entering: get_updates
2025-07-06 10:27:30,047 - httpcore.http11 - DEBUG - send_request_headers.started request=<Request [b'POST']>
2025-07-06 10:27:30,047 - httpcore.http11 - DEBUG - send_request_headers.complete
2025-07-06 10:27:30,049 - httpcore.http11 - DEBUG - send_request_body.started request=<Request [b'POST']>
2025-07-06 10:27:30,050 - httpcore.http11 - DEBUG - send_request_body.complete
2025-07-06 10:27:30,050 - httpcore.http11 - DEBUG - receive_response_headers.started request=<Request [b'POST']>
2025-07-06 10:27:34,169 - httpcore.http11 - DEBUG - receive_response_headers.complete return_value=(b'HTTP/1.1', 409, b'Conflict', [(b'Server', b'nginx/1.18.0'), (b'Date', b'Sun, 06 Jul 2025 09:27:34 GMT'), (b'Content-Type', b'application/json'), (b'Content-Length', b'143'), (b'Connection', b'keep-alive'), (b'Strict-Transport-Security', b'max-age=31536000; includeSubDomains; preload'), (b'Access-Control-Allow-Origin', b'*'), (b'Access-Control-Expose-Headers', b'Content-Length,Content-Type,Date,Server,Connection')])
2025-07-06 10:27:34,169 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot7707158318:AAE8rWa4Q8Z_3avql6K0HAmXaooZAReoNF0/getUpdates "HTTP/1.1 409 Conflict"
2025-07-06 10:27:34,171 - httpcore.http11 - DEBUG - receive_response_body.started request=<Request [b'POST']>
2025-07-06 10:27:34,171 - httpcore.http11 - DEBUG - receive_response_body.complete
2025-07-06 10:27:34,171 - httpcore.http11 - DEBUG - response_closed.started
2025-07-06 10:27:34,171 - httpcore.http11 - DEBUG - response_closed.complete
2025-07-06 10:27:34,172 - telegram.ext.Updater - ERROR - Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
2025-07-06 10:27:34,173 - telegram.ext.Application - ERROR - No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\ext\_updater.py", line 628, in _network_loop_retry
    if not await action_cb():
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\ext\_updater.py", line 333, in polling_action_cb
    raise exc
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\ext\_updater.py", line 322, in polling_action_cb
    updates = await self.bot.get_updates(
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\ext\_extbot.py", line 553, in get_updates
    updates = await super().get_updates(
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\_bot.py", line 394, in decorator
    result = await func(self, *args, **kwargs)  # skipcq: PYL-E1102
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\_bot.py", line 3542, in get_updates
    await self._post(
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\_bot.py", line 482, in _post
    return await self._do_post(
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\ext\_extbot.py", line 335, in _do_post
    return await super()._do_post(
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\_bot.py", line 510, in _do_post
    return await request.post(
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\request\_baserequest.py", line 168, in post
    result = await self._request_wrapper(
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\request\_baserequest.py", line 327, in _request_wrapper
    raise Conflict(message)
telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
2025-07-06 10:27:35,668 - telegram.ext.ExtBot - DEBUG - Entering: get_updates
2025-07-06 10:27:35,672 - httpcore.http11 - DEBUG - send_request_headers.started request=<Request [b'POST']>
2025-07-06 10:27:35,673 - httpcore.http11 - DEBUG - send_request_headers.complete
2025-07-06 10:27:35,673 - httpcore.http11 - DEBUG - send_request_body.started request=<Request [b'POST']>
2025-07-06 10:27:35,674 - httpcore.http11 - DEBUG - send_request_body.complete
2025-07-06 10:27:35,674 - httpcore.http11 - DEBUG - receive_response_headers.started request=<Request [b'POST']>
2025-07-06 10:27:40,283 - httpcore.http11 - DEBUG - receive_response_headers.complete return_value=(b'HTTP/1.1', 409, b'Conflict', [(b'Server', b'nginx/1.18.0'), (b'Date', b'Sun, 06 Jul 2025 09:27:40 GMT'), (b'Content-Type', b'application/json'), (b'Content-Length', b'143'), (b'Connection', b'keep-alive'), (b'Strict-Transport-Security', b'max-age=31536000; includeSubDomains; preload'), (b'Access-Control-Allow-Origin', b'*'), (b'Access-Control-Expose-Headers', b'Content-Length,Content-Type,Date,Server,Connection')])
2025-07-06 10:27:40,283 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot7707158318:AAE8rWa4Q8Z_3avql6K0HAmXaooZAReoNF0/getUpdates "HTTP/1.1 409 Conflict"
2025-07-06 10:27:40,284 - httpcore.http11 - DEBUG - receive_response_body.started request=<Request [b'POST']>
2025-07-06 10:27:40,284 - httpcore.http11 - DEBUG - receive_response_body.complete
2025-07-06 10:27:40,284 - httpcore.http11 - DEBUG - response_closed.started
2025-07-06 10:27:40,284 - httpcore.http11 - DEBUG - response_closed.complete
2025-07-06 10:27:40,284 - telegram.ext.Updater - ERROR - Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
2025-07-06 10:27:40,285 - telegram.ext.Application - ERROR - No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\ext\_updater.py", line 628, in _network_loop_retry
    if not await action_cb():
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\ext\_updater.py", line 333, in polling_action_cb
    raise exc
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\ext\_updater.py", line 322, in polling_action_cb
    updates = await self.bot.get_updates(
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\ext\_extbot.py", line 553, in get_updates
    updates = await super().get_updates(
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\_bot.py", line 394, in decorator
    result = await func(self, *args, **kwargs)  # skipcq: PYL-E1102
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\_bot.py", line 3542, in get_updates
    await self._post(
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\_bot.py", line 482, in _post
    return await self._do_post(
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\ext\_extbot.py", line 335, in _do_post
    return await super()._do_post(
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\_bot.py", line 510, in _do_post
    return await request.post(
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\request\_baserequest.py", line 168, in post
    result = await self._request_wrapper(
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\request\_baserequest.py", line 327, in _request_wrapper
    raise Conflict(message)
telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
2025-07-06 10:27:42,549 - telegram.ext.ExtBot - DEBUG - Entering: get_updates
2025-07-06 10:27:42,550 - httpcore.http11 - DEBUG - send_request_headers.started request=<Request [b'POST']>
2025-07-06 10:27:42,551 - httpcore.http11 - DEBUG - send_request_headers.complete
2025-07-06 10:27:42,552 - httpcore.http11 - DEBUG - send_request_body.started request=<Request [b'POST']>
2025-07-06 10:27:42,552 - httpcore.http11 - DEBUG - send_request_body.complete
2025-07-06 10:27:42,552 - httpcore.http11 - DEBUG - receive_response_headers.started request=<Request [b'POST']>
2025-07-06 10:27:47,902 - httpcore.http11 - DEBUG - receive_response_headers.complete return_value=(b'HTTP/1.1', 409, b'Conflict', [(b'Server', b'nginx/1.18.0'), (b'Date', b'Sun, 06 Jul 2025 09:27:48 GMT'), (b'Content-Type', b'application/json'), (b'Content-Length', b'143'), (b'Connection', b'keep-alive'), (b'Strict-Transport-Security', b'max-age=31536000; includeSubDomains; preload'), (b'Access-Control-Allow-Origin', b'*'), (b'Access-Control-Expose-Headers', b'Content-Length,Content-Type,Date,Server,Connection')])
2025-07-06 10:27:47,904 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot7707158318:AAE8rWa4Q8Z_3avql6K0HAmXaooZAReoNF0/getUpdates "HTTP/1.1 409 Conflict"
2025-07-06 10:27:47,905 - httpcore.http11 - DEBUG - receive_response_body.started request=<Request [b'POST']>
2025-07-06 10:27:47,906 - httpcore.http11 - DEBUG - receive_response_body.complete
2025-07-06 10:27:47,906 - httpcore.http11 - DEBUG - response_closed.started
2025-07-06 10:27:47,906 - httpcore.http11 - DEBUG - response_closed.complete
2025-07-06 10:27:47,906 - telegram.ext.Updater - ERROR - Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
2025-07-06 10:27:47,907 - telegram.ext.Application - ERROR - No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\ext\_updater.py", line 628, in _network_loop_retry
    if not await action_cb():
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\ext\_updater.py", line 333, in polling_action_cb
    raise exc
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\ext\_updater.py", line 322, in polling_action_cb
    updates = await self.bot.get_updates(
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\ext\_extbot.py", line 553, in get_updates
    updates = await super().get_updates(
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\_bot.py", line 394, in decorator
    result = await func(self, *args, **kwargs)  # skipcq: PYL-E1102
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\_bot.py", line 3542, in get_updates
    await self._post(
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\_bot.py", line 482, in _post
    return await self._do_post(
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\ext\_extbot.py", line 335, in _do_post
    return await super()._do_post(
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\_bot.py", line 510, in _do_post
    return await request.post(
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\request\_baserequest.py", line 168, in post
    result = await self._request_wrapper(
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\request\_baserequest.py", line 327, in _request_wrapper
    raise Conflict(message)
telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
2025-07-06 10:27:51,277 - telegram.ext.ExtBot - DEBUG - Entering: get_updates
2025-07-06 10:27:51,280 - httpcore.http11 - DEBUG - send_request_headers.started request=<Request [b'POST']>
2025-07-06 10:27:51,281 - httpcore.http11 - DEBUG - send_request_headers.complete
2025-07-06 10:27:51,281 - httpcore.http11 - DEBUG - send_request_body.started request=<Request [b'POST']>
2025-07-06 10:27:51,281 - httpcore.http11 - DEBUG - send_request_body.complete
2025-07-06 10:27:51,281 - httpcore.http11 - DEBUG - receive_response_headers.started request=<Request [b'POST']>
2025-07-06 10:27:54,762 - httpcore.http11 - DEBUG - receive_response_headers.complete return_value=(b'HTTP/1.1', 409, b'Conflict', [(b'Server', b'nginx/1.18.0'), (b'Date', b'Sun, 06 Jul 2025 09:27:55 GMT'), (b'Content-Type', b'application/json'), (b'Content-Length', b'143'), (b'Connection', b'keep-alive'), (b'Strict-Transport-Security', b'max-age=31536000; includeSubDomains; preload'), (b'Access-Control-Allow-Origin', b'*'), (b'Access-Control-Expose-Headers', b'Content-Length,Content-Type,Date,Server,Connection')])
2025-07-06 10:27:54,762 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot7707158318:AAE8rWa4Q8Z_3avql6K0HAmXaooZAReoNF0/getUpdates "HTTP/1.1 409 Conflict"
2025-07-06 10:27:54,763 - httpcore.http11 - DEBUG - receive_response_body.started request=<Request [b'POST']>
2025-07-06 10:27:54,764 - httpcore.http11 - DEBUG - receive_response_body.complete
2025-07-06 10:27:54,764 - httpcore.http11 - DEBUG - response_closed.started
2025-07-06 10:27:54,764 - httpcore.http11 - DEBUG - response_closed.complete
2025-07-06 10:27:54,764 - telegram.ext.Updater - ERROR - Error while getting Updates: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
2025-07-06 10:27:54,765 - telegram.ext.Application - ERROR - No error handlers are registered, logging exception.
Traceback (most recent call last):
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\ext\_updater.py", line 628, in _network_loop_retry
    if not await action_cb():
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\ext\_updater.py", line 333, in polling_action_cb
    raise exc
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\ext\_updater.py", line 322, in polling_action_cb
    updates = await self.bot.get_updates(
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\ext\_extbot.py", line 553, in get_updates
    updates = await super().get_updates(
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\_bot.py", line 394, in decorator
    result = await func(self, *args, **kwargs)  # skipcq: PYL-E1102
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\_bot.py", line 3542, in get_updates
    await self._post(
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\_bot.py", line 482, in _post
    return await self._do_post(
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\ext\_extbot.py", line 335, in _do_post
    return await super()._do_post(
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\_bot.py", line 510, in _do_post
    return await request.post(
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\request\_baserequest.py", line 168, in post
    result = await self._request_wrapper(
  File "c:\Users\Rachid Aitali\OneDrive - 4JM Solutions Limited\Desktop\SniperX-V3-Best-Version-at-now\.venv\lib\site-packages\telegram\request\_baserequest.py", line 327, in _request_wrapper
    raise Conflict(message)
telegram.error.Conflict: Conflict: terminated by other getUpdates request; make sure that only one bot instance is running
2025-07-06 10:27:59,840 - telegram.ext.ExtBot - DEBUG - Entering: get_updates
2025-07-06 10:27:59,842 - httpcore.connection - DEBUG - close.started
2025-07-06 10:27:59,842 - httpcore.connection - DEBUG - close.complete
2025-07-06 10:27:59,842 - httpcore.connection - DEBUG - connect_tcp.started host='api.telegram.org' port=443 local_address=None timeout=5.0 socket_options=None
2025-07-06 10:27:59,902 - httpcore.connection - DEBUG - connect_tcp.complete return_value=<httpcore._backends.anyio.AnyIOStream object at 0x000001BB99BA1C60>
2025-07-06 10:27:59,902 - httpcore.connection - DEBUG - start_tls.started ssl_context=<ssl.SSLContext object at 0x000001BB989A8240> server_hostname='api.telegram.org' timeout=5.0
2025-07-06 10:27:59,960 - httpcore.connection - DEBUG - start_tls.complete return_value=<httpcore._backends.anyio.AnyIOStream object at 0x000001BB99BA1D50>
2025-07-06 10:27:59,960 - httpcore.http11 - DEBUG - send_request_headers.started request=<Request [b'POST']>
2025-07-06 10:27:59,960 - httpcore.http11 - DEBUG - send_request_headers.complete
2025-07-06 10:27:59,961 - httpcore.http11 - DEBUG - send_request_body.started request=<Request [b'POST']>
2025-07-06 10:27:59,961 - httpcore.http11 - DEBUG - send_request_body.complete
2025-07-06 10:27:59,961 - httpcore.http11 - DEBUG - receive_response_headers.started request=<Request [b'POST']>
2025-07-06 10:28:10,010 - httpcore.http11 - DEBUG - receive_response_headers.complete return_value=(b'HTTP/1.1', 200, b'OK', [(b'Server', b'nginx/1.18.0'), (b'Date', b'Sun, 06 Jul 2025 09:28:10 GMT'), (b'Content-Type', b'application/json'), (b'Content-Length', b'23'), (b'Connection', b'keep-alive'), (b'Strict-Transport-Security', b'max-age=31536000; includeSubDomains; preload'), (b'Access-Control-Allow-Origin', b'*'), (b'Access-Control-Allow-Methods', b'GET, POST, OPTIONS'), (b'Access-Control-Expose-Headers', b'Content-Length,Content-Type,Date,Server,Connection')])
2025-07-06 10:28:10,010 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot7707158318:AAE8rWa4Q8Z_3avql6K0HAmXaooZAReoNF0/getUpdates "HTTP/1.1 200 OK"
2025-07-06 10:28:10,010 - httpcore.http11 - DEBUG - receive_response_body.started request=<Request [b'POST']>
2025-07-06 10:28:10,010 - httpcore.http11 - DEBUG - receive_response_body.complete
2025-07-06 10:28:10,010 - httpcore.http11 - DEBUG - response_closed.started
2025-07-06 10:28:10,010 - httpcore.http11 - DEBUG - response_closed.complete
2025-07-06 10:28:10,010 - telegram.ext.ExtBot - DEBUG - No new updates found.
2025-07-06 10:28:10,010 - telegram.ext.ExtBot - DEBUG - ()
2025-07-06 10:28:10,010 - telegram.ext.ExtBot - DEBUG - Exiting: get_updates
2025-07-06 10:28:10,013 - telegram.ext.ExtBot - DEBUG - Entering: get_updates
2025-07-06 10:28:10,013 - httpcore.http11 - DEBUG - send_request_headers.started request=<Request [b'POST']>
2025-07-06 10:28:10,015 - httpcore.http11 - DEBUG - send_request_headers.complete
2025-07-06 10:28:10,015 - httpcore.http11 - DEBUG - send_request_body.started request=<Request [b'POST']>
2025-07-06 10:28:10,015 - httpcore.http11 - DEBUG - send_request_body.complete
2025-07-06 10:28:10,017 - httpcore.http11 - DEBUG - receive_response_headers.started request=<Request [b'POST']>
2025-07-06 10:28:20,075 - httpcore.http11 - DEBUG - receive_response_headers.complete return_value=(b'HTTP/1.1', 200, b'OK', [(b'Server', b'nginx/1.18.0'), (b'Date', b'Sun, 06 Jul 2025 09:28:20 GMT'), (b'Content-Type', b'application/json'), (b'Content-Length', b'23'), (b'Connection', b'keep-alive'), (b'Strict-Transport-Security', b'max-age=31536000; includeSubDomains; preload'), (b'Access-Control-Allow-Origin', b'*'), (b'Access-Control-Allow-Methods', b'GET, POST, OPTIONS'), (b'Access-Control-Expose-Headers', b'Content-Length,Content-Type,Date,Server,Connection')])
2025-07-06 10:28:20,076 - httpx - INFO - HTTP Request: POST https://api.telegram.org/bot7707158318:AAE8rWa4Q8Z_3avql6K0HAmXaooZAReoNF0/getUpdates "HTTP/1.1 200 OK"
2025-07-06 10:28:20,077 - httpcore.http11 - DEBUG - receive_response_body.started request=<Request [b'POST']>
2025-07-06 10:28:20,077 - httpcore.http11 - DEBUG - receive_response_body.complete
2025-07-06 10:28:20,077 - httpcore.http11 - DEBUG - response_closed.started
2025-07-06 10:28:20,077 - httpcore.http11 - DEBUG - response_closed.complete
2025-07-06 10:28:20,077 - telegram.ext.ExtBot - DEBUG - No new updates found.
2025-07-06 10:28:20,077 - telegram.ext.ExtBot - DEBUG - ()
2025-07-06 10:28:20,077 - telegram.ext.ExtBot - DEBUG - Exiting: get_updates
2025-07-06 10:28:20,077 - telegram.ext.ExtBot - DEBUG - Entering: get_updates
2025-07-06 10:28:20,077 - httpcore.http11 - DEBUG - send_request_headers.started request=<Request [b'POST']>
2025-07-06 10:28:20,077 - httpcore.http11 - DEBUG - send_request_headers.complete
2025-07-06 10:28:20,077 - httpcore.http11 - DEBUG - send_request_body.started request=<Request [b'POST']>
2025-07-06 10:28:20,080 - httpcore.http11 - DEBUG - send_request_body.complete
2025-07-06 10:28:20,080 - httpcore.http11 - DEBUG - receive_response_headers.started request=<Request [b'POST']>
